charts/gardener/controlplane/charts/runtime/templates/scheduler/deployment.yaml

```diff
@@ -71,8 +71,7 @@ spec:
       - name: gardener-scheduler
         image: {{ include "utils-templates.image" .Values.global.scheduler.image }}
         imagePullPolicy: {{ .Values.global.scheduler.image.pullPolicy }}
-        command:
-        - /gardener-scheduler
```

Comment:
If I understand it correctly you want to rely on what is set as a default `ENTRYPOINT` for the image. In this way if we change the `ENTRYPOINT` during image build we do not have to change it in several other places as well. Is this correct?

Reply:
That's exactly my motivation, yes.